### PR TITLE
Only return results of visits that have been updated in the last six months

### PR DESCRIPTION
**Pull request by TobyRet**
https://trello.com/c/0E58iabh/1428-only-show-data-from-last-6-months-in-search-results

### CODE REVIEW

File: spec/services/estate_visit_query_spec.rb [hunk 1]
```diff
@@ -151,9 +170,14 @@
       FactoryBot.create(:visit, :requested, prison: prison)
     end
 
+    let!(:old_visit) do
+      FactoryBot.create(:visit, :requested, prison: prison, created_at: 7.months.ago, updated_at: 7.months.ago)
```
Comment (StupidCodeFactory) [inline spec/services/estate_visit_query_spec.rb]: no need to use `FactoryBot` just call `create(:visit)`

File: spec/services/estate_visit_query_spec.rb [hunk 2]
```diff
@@ -106,6 +106,25 @@
           is_expected.to eq([booked])
         end
       end
+
+      context 'when visits have not been updated within six months' do
+        let!(:old_booked) { FactoryBot.create(:booked_visit, prison: prison, updated_at: 7.months.ago) }
+        let(:prisoner_number) { old_booked.prisoner.number.downcase + ' ' }
```
Comment (StupidCodeFactory) [inline spec/services/estate_visit_query_spec.rb]: is the `+ ' '` to see if the class trims the input?

File: spec/services/estate_visit_query_spec.rb (hunk 1 above)
Comment (StupidCodeFactory) [reply spec/services/estate_visit_query_spec.rb]: basically if you see `FactoryBot` in front of a `create` or `build` or `build_stubbed` and so on rip it off!

File: spec/services/estate_visit_query_spec.rb (hunk 2 above)
Comment (TobyRet) [reply spec/services/estate_visit_query_spec.rb]: @StupidCodeFactory - I don't think the trailing white space is required. I've removed both instances and tests pass.

**Pull request merged**